components/script/dom/htmlareaelement.rs

```diff
@@ -53,8 +56,8 @@ pub enum Shape {
     Polygon,
 }
 
-// https://html.spec.whatwg.org/multipage/#rules-for-parsing-a-list-of-floating-point-numbers
-// https://html.spec.whatwg.org/multipage/#image-map-processing-model
+/// <https://html.spec.whatwg.org/multipage/#rules-for-parsing-a-list-of-floating-point-numbers>
+/// <https://html.spec.whatwg.org/multipage/#image-map-processing-model>
```

Comment:
Are doc comments on impl blocks a thing? Shouldn't these be on one of the `fn` blocks?

Reply:
Ah good catch, I have reverted this change 👍 Thank you.